### DISCUSSION TITLE
How do I update (docker compose)

### DISCUSSION THREAD

**Discussion by ETechTechnologies** [ArchiveBox/ArchiveBox#815] [Q&A]
I'm using docker-compose. How do I update archivebox? Current version: v0.6.2

**Comment by pirate** [comment]
It can be as simple as `docker-compose down; docker-compose pull; docker-compose up` but I want to make sure you don't have a custom `command:` line that might prevent the auto-updating migrations feature from running.

Can you post your `docker-compose.yml` file contents?

**Reply by ETechTechnologies** [reply]
Where is that file located?

**Reply by pirate** [reply]
It's in the root of wherever you put the docker-compose archivebox directory when you first set it up. The archivebox directory is chosen by the user, not something the software dictates. One common location used in tutorials is `~/archivebox`.
